Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -47,22 +47,26 @@ public final class WalletManager: Equatable {
     /// transfer (like BRD transfer => ETH fee)
     public lazy var primaryWallet: Wallet = {
         // Find a preexisting wallet (unlikely) or create one.
-        return Wallet (core: cryptoWalletManagerGetWallet(core),
+        let coreWallet = cryptoWalletManagerGetWallet(core)!
+        return Wallet (core: coreWallet,
                        listener: system.listener,
                        manager: self,
-                       unit: unit,
```

Comment:
Purely a question but why not have the cryptoWalletGetUnit() call in the ctor?

Reply:
Fixed.